src/tests/remote.test.ts

```diff
@@ -64,4 +81,33 @@ describe('Perl debugger connection', () => {
 		assert.equal(res.ln, 7); // The first code line in test.pl is 7
 	});
 
+	it('Should be able to get loaded scripts and their source code from' + FILE_TEST_PL, async () => {
+
+		const [ server, local ] = setupDebugger(
+			conn, FILE_TEST_PL, DATA_ROOT, [], launchOptions
+		);
+
+		// Wait for result
+		const res = await server;
+
+		const loadedFiles = await conn.getLoadedFiles();
+		const modulePms = loadedFiles.filter(
+			x => x.endsWith('Module.pm')
+		);
+
+		assert.ok(
+			modulePms.length > 0,
+			'Must have loaded a `Module.pm` file'
+		);
```

Comment:
we are using jest so we could use `expect(modulePms).toMatchSnapshot`
https://jestjs.io/docs/en/snapshot-testing

Reply:
Not here, file system paths likely vary based on the environment the test is running in.

Reply:
good point